components/navbar/navbar.css

```diff
@@ -55,8 +55,7 @@
 }
 
 .ampstart-navbar-trigger {
-  line-height: var(--navbar-height);
-  font-size: var(--h2);
+  height: 24px;
```

Comment:
Menu icon is no longer a character but instead an icon font. This was because changes to the font color were not adhered to with the Samsung S6. Changing to an SVG will fill:currentColor ensures the icon is not only consistent across all devices but can also be styled.